controllers/setup.go

```diff
@@ -50,6 +50,19 @@ func CreateAndSetupReconciler(mgr controllerruntime.Manager) error {
 		return err
 	}
 
+	serviceController := CreateServiceController(mgr)
+
+	mgr.Add(manager.RunnableFunc(func(ctx context.Context) error {
```

Comment:
Can you check the error returned by `mgr.Add()`?

Reply:
I can, and I should :) added.